ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -203,7 +204,8 @@ tests:
     cluster_profile: packet-assisted
     env:
       POST_INSTALL_COMMANDS: |
-        export TEST_FUNC=test_kube_api_ipv4
+        export TEST_FUNC=test_kubeapi
```

Comment:
Same as above. system-test-operator is not indicative

Reply:
Will remove it